src/apiMethods.js

```diff
@@ -0,0 +1,24 @@
+const addScore = async (username, score) => {
+  await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/RbZstIDnGYdGTsk3sKDW/scores', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      user: username,
+      score,
+    }),
+  });
+};
+
+const getScores = async () => (
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/RbZstIDnGYdGTsk3sKDW/scores', {
+    method: 'GET',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  }).then((response) => response.json())
+    .then((data) => data.result)
+);
```

Comment:
- [ ] Great job so far 👍 your App displays well and works as expected but in the project requirements you are to use `async` and `await` JavaScript features to consume the API but you have implemented your features using `promises`. Kindly rectify this issue in all the cases where you are fetching data. Below is an implemented example:
- Async function:

![Annotation 2022-03-02 161448](https://user-images.githubusercontent.com/85113567/156390118-9761b0bc-b387-4542-b514-aafdfa7cc8aa.jpg)